pkg/volume/cinder/cinder.go

```diff
@@ -89,7 +89,7 @@ func (plugin *cinderPlugin) newBuilderInternal(spec *volume.Spec, podUID types.U
 		},
 		fsType:             fsType,
 		readOnly:           readOnly,
-		blockDeviceMounter: &cinderSafeFormatAndMount{mounter, exec.New()}}, nil
+		blockDeviceMounter: &mount.SafeFormatAndMount{plugin.host.GetMounter(), exec.New()}}, nil
```

Comment:
Hmm.. s/plugin.host.GetMounter()/mounter/
Not sure if the cinder tests use this but other volume plugin tests use this to pass in mock arguments